### ISSUE TITLE
Commits graph view be shown by week instead of month

### ISSUE THREAD

**Issue by duffybelfield** [morucci/repoxplorer#52]
Is there a way to show the commit graph by week instead of by Month?

**Comment by morucci** [comment]
No the scaling is automatic. See https://github.com/morucci/repoxplorer/blob/master/repoxplorer/index/commits.py#L685. But let's set that issue as RFE.
Do you expect to have a choice in the UI (cumulate by day/week/month/year) ?

Thanks for the report.

**Comment by duffybelfield** [comment]
Yes in the UI would be great and then scrollable? Could you also set if you want to use Week / Month in the configuration?

**Comment by morucci** [comment]
Yes the project configuration could provide a setting to set the default histogram scaling and then have the choice in the UI (the default might not be good for every repo). Scrollable would be awesome but I lack JS/UI skills to implements that. The plan for the future is to rewrite the JS part with react and I assume it is going to be easier than with jquery.
